src/drivers/gps/params.c

```diff
@@ -87,27 +87,45 @@ PARAM_DEFINE_INT32(GPS_SAT_INFO, 0);
  * Select the u-blox configuration setup. Most setups will use the default, including RTK and
  * dual GPS without heading.
  *
+ * If rover has RTCM corrections from a static base (or other static correction source) coming in on UART2, then select Mode 1.
  * The Heading mode requires 2 F9P devices to be attached. The main GPS will act as rover and output
  * heading information, whereas the secondary will act as moving base.
- * Modes 1 and 2 require each F9P UART1 to be connected to the Autopilot. In addition, UART2 on the
+ * Modes 2 and 3 require each F9P UART1 to be connected to the Autopilot. In addition, UART2 on the
  * F9P units are connected to each other.
- * Modes 3 and 4 only require UART1 on each F9P connected to the Autopilot or Can Node. UART RX DMA is required.
+ * Modes 4 and 5 only require UART1 on each F9P connected to the Autopilot or Can Node. UART RX DMA is required.
  * RTK is still possible with this setup.
  *
  * @min 0
  * @max 1
  * @value 0 Default
- * @value 1 Heading (Rover With Moving Base UART1 Connected To Autopilot, UART2 Connected To Moving Base)
- * @value 2 Moving Base (UART1 Connected To Autopilot, UART2 Connected To Rover)
- * @value 3 Heading (Rover With Moving Base UART1 Connected to Autopilot Or Can Node At 921600)
- * @value 4 Moving Base (Moving Base UART1 Connected to Autopilot Or Can Node At 921600)
+ * @value 1 Rover with Static Base on UART2 (similar to Default, except coming in on UART2)
```

Comment:
Changing the values breaks existing setups.
I think you can reorder the list here though to change the display order.

Reply:
QGC didn't order it correctly. I kept the existing values so as not to break existing setups, and assigned my new mode a value of 5.